### PR TITLE
Critical/Delivery Time not really a ServiceInsight concern

### PR DESCRIPTION
**Pull request by weralabaj**
Remove Critical Time and Delivery Time from:
- messages table
- flow diagram message tooltip
- message properties
- related view models and data objects.

### CODE REVIEW

File: src/ServiceInsight/Models/StoredMessage.cs [hunk 1]
```diff
@@ -19,9 +19,7 @@ public StoredMessage()
         public MessageIntent MessageIntent { get; set; }
         public Endpoint SendingEndpoint { get; set; }
         public Endpoint ReceivingEndpoint { get; set; }
-        public TimeSpan CriticalTime { get; set; }
         public TimeSpan ProcessingTime { get; set; }
-        public TimeSpan DeliveryTime { get; set; }
         public string ConversationId { get; set; }
 
         public MessageStatistics Statistics
```
Comment (weralabaj) [inline src/ServiceInsight/Models/StoredMessage.cs]: @distantcam I think that this class is not used anywhere at the moment, can we get rid of it or did I miss something?

Comment (johnsimons) [inline src/ServiceInsight/Models/StoredMessage.cs]: If it is not in use, u can delete it

**Comment by weralabaj** [comment]
@johnsimons It doesn't seem to be but then if you look at the [history](https://github.com/Particular/ServiceInsight/commits/d039dca8eabe0641041ca3d2b7b48f04b2d785d7/src/ServiceInsight.Desktop/Models/MessageStatistics.cs) it was removed and then added again. Weird. So I wanted to double check if I missed anything.

**Comment by weralabaj** [comment]
Visual summary of the change, @johnsimons or @distantcam please see if that's good to go.
critical_delivery_times_in_si

**Comment by johnsimons** [comment]
Did u also remove it from the sequence diagram?

**Comment by weralabaj** [comment]
From code yes, but I don't see how it's displayed so don't have a screen to show change (is there a bug or hidden feature that I haven't found?).

**Comment by johnsimons** [comment]
I thought u just hover over the rectangle areas, it's in a tooltip.

**Comment by weralabaj** [comment]
Hm this is what I've seen even before my changes:
sd
Should there be more?

**Comment by johnsimons** [comment]
Tooltip when u hover over the grey rectangle on the line.
Anyway I can review it on Monday.

**Comment by weralabaj** [comment]
Thanks @johnsimons, I didn't think about that one! Updated visual summary and adjusted height of tooltip.

**Comment by johnsimons** [comment]
:+1: to merge

**Comment by weralabaj** [comment]
@johnsimons Should I merge it myself? Or is there some flow and caretaker/somebody else does it?

**Comment by johnsimons** [comment]
Go for it, you can do it ;)

On Tue, 28 Jul 2015 at 17:39 Weronika Łabaj <notifications@github.com>
wrote:

> @johnsimons <https://github.com/johnsimons> Should I merge it myself? Or
> is there some flow and caretaker/somebody else does it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/ServiceInsight/pull/477#issuecomment-125484572>
> .
>
-- 

*John Simons*
*Engineer, Particular Software*

Email*:*  john.simons@particular.net <karen.fruchtman@particular.net>
Web:    www.particular.net

**Pull request merged**